1 Niagara 4 Overview Exercise - BFile/myFile-rt/src/com/tridiumuniversity/myFile/BFile.java

```diff
@@ -0,0 +1,240 @@
+package com.tridiumuniversity.myFile;
+
+import javax.baja.nre.annotations.NiagaraAction;
+import javax.baja.nre.annotations.NiagaraProperty;
+import javax.baja.nre.annotations.NiagaraType;
+import javax.baja.status.BStatus;
+import javax.baja.status.BStatusNumeric;
+import javax.baja.status.BStatusString;
+import javax.baja.sys.Action;
+import javax.baja.sys.BAbsTime;
+import javax.baja.sys.BComponent;
+import javax.baja.sys.BMonth;
+import javax.baja.sys.Flags;
+import javax.baja.sys.Property;
+import javax.baja.sys.Slot;
+import javax.baja.sys.Sys;
+import javax.baja.sys.Type;
+
+@NiagaraType
+@NiagaraProperty(
+        name = "fileName",
+        type = "BStatusString",
+        defaultValue = "new BStatusString (\"\", BStatus.nullStatus)",
+        flags = Flags.SUMMARY
+)
+
+@NiagaraProperty(
+        name = "path",
+        type = "String",
+        defaultValue = "\"path\"",
```

Comment:
This is fine, but you don't have to include the escaped quotes. You could just use: `defaultValue = "path"`